### PR TITLE
ENH: Remove unnecessary logging information from ctkDICOMDatabase class

### PR DESCRIPTION
**Pull request by Punzo**
Fix https://github.com/commontk/CTK/issues/1186

### CODE REVIEW

**Comment by Punzo** [comment]
@lassoan @jcfr

File: Libs/DICOM/Core/ctkDICOMDatabase.cpp [hunk 1]
```diff
@@ -159,16 +159,6 @@ bool ctkDICOMDatabasePrivate::loggedExec(QSqlQuery& query, const QString& queryS
   {
     success = query.exec();
   }
-  if (!success)
-  {
-    QSqlError sqlError = query.lastError();
-    logger.debug("SQL failed\n Bad SQL: " + query.lastQuery());
```
Comment (lassoan) [inline Libs/DICOM/Core/ctkDICOMDatabase.cpp:165]: Errors should be still logged, probably at least at a warning level (unless there are a lot of false alarms, but I don't think there should be any).

File: Libs/DICOM/Core/ctkDICOMDatabase.cpp [hunk 2]
```diff
@@ -851,13 +828,11 @@ void ctkDICOMDatabasePrivate::insert(const ctkDICOMItem& dataset, const QString&
   {
     if (datasetUpToDate)
     {
-      logger.debug("File " + databaseFilename + " already added");
```
Comment (lassoan) [inline Libs/DICOM/Core/ctkDICOMDatabase.cpp:854]: Keep this, rarely happens and useful for troubleshooting

File: Libs/DICOM/Core/ctkDICOMDatabase.cpp [hunk 3]
```diff
@@ -851,13 +828,11 @@ void ctkDICOMDatabasePrivate::insert(const ctkDICOMItem& dataset, const QString&
...
       return;
     }
     // File is updated, delete record and re-index
     if (!this->removeImage(sopInstanceUID))
     {
-      logger.debug("File " + filePath + " cannot be added, failed to update existing values in the database");
```
Comment (lassoan) [inline Libs/DICOM/Core/ctkDICOMDatabase.cpp:860]: Keep this, rarely happens and useful for troubleshooting

File: Libs/DICOM/Core/ctkDICOMDatabase.cpp [hunk 4]
```diff
@@ -2373,12 +2346,9 @@ void ctkDICOMDatabase::insert( const QString& filePath, bool storeFile, bool gen
   /// first we check if the file is already in the database
   if (fileExistsAndUpToDate(filePath))
   {
-    logger.debug( "File " + filePath + " already added.");
```
Comment (lassoan) [inline Libs/DICOM/Core/ctkDICOMDatabase.cpp:2376]: Keep this, rarely happens and useful for troubleshooting

**Comment by Punzo** [comment]
> I would make some adjustments (see comments inline) but overall I like this simple solution: if something is important for user to know or useful for troubleshooting (and does not happen to frequently) then log the message, otherwise remove the log.

ok and thanks for the review. I have applied it! @lassoan

**Review by lassoan — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**